src/coreclr/src/pal/src/exception/machexception.cpp

```diff
@@ -441,12 +446,36 @@ BuildExceptionRecord(
         }
         else
         {
+#if defined(HOST_AMD64) || defined(HOST_X86)
```

Comment:
x86 is not supported (there are more checks for x86 below, I'll stop commenting on them)